### ISSUE TITLE
Horizon: Zero Dawn doesn't launch

### ISSUE THREAD

**Issue by DrSay123** [ptitSeb/box64#1047]
Wine: 8.2 staging/8.14 ge. Both has this error
When I'm trying to launch this game, its appear in taskmgr for few seconds and disappear. No error, no game's window. I installed vkd3d-proton, vcredist and compiled latest version of box64 with sse 4.2 instructions, but there was no help
[debug.log](https://github.com/ptitSeb/box64/files/13227376/debug.log)

**Comment by DrSay123** [comment]
> I'm downloading the GoG version, will probably try it in a few days...

**Comment by DrSay123** [comment]
So, how was it? Did you test it out?

**Comment by DrSay123** [comment]
So, how was it? Did you test it out?

**Comment by ptitSeb** [comment]
Yeah, I tested it tonight. I reproduce a crash. I tested a few things, and it's not a dynarec issue, as the same crash is present without it. I'll continue to investigate tomorrow (I'm using the GoG Complete Edition).

On my side (after adding a `BOX64_DYNAREC_STRONGMEM=1`), I have loading screen, selection of language and subtitle, got the SONY logo, but then I have a crash when I move the mouse....

**Comment by DrSay123** [comment]
I just rechecked this game today - it runs for now, but, as you said, crash on menu/loading
Did you find why it happens?

**Comment by ruje0504** [comment]
I successfully ran it using Wine9.0
【AYN 奥丁2 archlinux box64 wine vkd3d 地平线：零之曙光-哔哩哔哩】 https://b23.tv/Gl60O1c

8gen2
The cpu is under a lot of load.
Versions below  wine9.0 will receive a crash notification in the game interface.

**Comment by ptitSeb** [comment]
Oooh, interesting, I'll go recheck this one later :)

**Comment by ruje0504** [comment]
> Oooh, interesting, I'll go recheck this one later :)

wine9.0 can only enter the game to play for a while, or occasionally crash

**Comment by LukeShortCloud** [comment]
@DrSay123 @ruje0504 is this still an issue with the latest Box64?
